projects/components/src/list-view/list-view.component.scss

```diff
@@ -7,6 +7,33 @@
   display: flex;
   flex-direction: column;
 
+  .header {
+    flex: 0 0 auto;
+    height: 32px;
+    width: 100%;
+    padding: 8px 0;
+    background: $gray-1;
+    box-shadow: inset 0px -1px 0px $gray-2;
+    border-radius: 6px 6px 0px 0px;
+    display: flex;
+    align-items: center;
+
+    .header-key {
+      flex: 1 1 auto;
+      font-weight: 600;
```

Comment:
Is there a mixin that we can use here?

Reply:
I was looking for it but could not find.
I will check more